examples/mysql/helper.conf

```diff
@@ -0,0 +1,19 @@
+# SPIRE agent unix socket path
+agentAddress = "/tmp/agent.sock"
+
+# mysql binary path
+cmd = "/usr/bin/mysql"
+
+# Query for configuration reloading
+cmdArgs = "-e \"ALTER INSTANCE RELOAD TLS;\""
```

Comment:
is this the only way to reloadcertificates? can you add comments about another options?